src/mappings/entities/mapConfidentialAccountCreated.ts

```diff
@@ -0,0 +1,26 @@
+import { ConfidentialAccount, EventIdEnum, ModuleIdEnum } from '../../types';
+import { getTextValue } from '../util';
+import { HandlerArgs } from './common';
+
+export const mapConfidentialAccountCreated = async (args: HandlerArgs): Promise<void> => {
+  const { blockId, moduleId, eventId, params } = args;
+
+  if (moduleId !== ModuleIdEnum.confidentialasset) {
+    return;
+  }
+
+  if (eventId !== EventIdEnum.AccountCreated) {
+    return;
+  }
```

Comment:
```suggestion
  if (moduleId !== ModuleIdEnum.confidentialasset || eventId !== EventIdEnum.AccountCreated) {
    return;
  }
```
nit -> both can be merged